src/contracts/core/DelegationManager.sol

```diff
@@ -550,16 +550,18 @@ contract DelegationManager is
 
         uint256[] memory prevSlashingFactors;
         {
-            uint32 completableBlock = withdrawal.startBlock + MIN_WITHDRAWAL_DELAY_BLOCKS;
-            require(completableBlock <= uint32(block.number), WithdrawalDelayNotElapsed());
+            // slashableUntil is block inclusive so we need to check if the current block is strictly greater than the slashableUntil block
+            // meaning the withdrawal can be completed.
+            uint32 slashableUntil = withdrawal.startBlock + MIN_WITHDRAWAL_DELAY_BLOCKS;
+            require(slashableUntil < uint32(block.number), WithdrawalDelayNotElapsed());
```

Comment:
`test_Revert_WhenWithdrawalDelayNotPassed` is a regression here, so we're good (already had this unit test-case covered)

Reply:
https://github.com/Layr-Labs/eigenlayer-contracts/pull/965/commits/6ffb6c399498cad8fc13223a95a28cf398fb3712